src/OrchardCore.Modules/OrchardCore.Deployment.Remote/OrchardCore.Deployment.Remote.csproj

```diff
@@ -3,6 +3,12 @@
   <PropertyGroup>
     <TargetFramework>$(AspNetCoreTargetFramework)</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+      <!-- NuGet properties-->
+    <Title>OrchardCore Deployment Remote</Title>
```

Comment:
"Depolyment Remote" sounds a bit funny. Should it be "OrchardCore Remote Deployment" instead?

Reply:
done